website/content/sdk/guides/intros/stocks/stocks-screener.md

```diff
@@ -0,0 +1,314 @@
+```
+
+|    | Ticker   | Company                               | Sector                 | Industry                       | Country   |   Market Cap |    P/E |   Price |   Change |           Volume |
+|---:|:---------|:--------------------------------------|:-----------------------|:-------------------------------|:----------|-------------:|-------:|--------:|---------:|-----------------:|
+|  0 | QFIN     | 360 DigiTech, Inc.                    | Financial              | Credit Services                | China     |    2.19e+09  |   3.73 |   14.56 |   0.0849 |      1.61386e+06 |
+|  1 | ATHM     | Autohome Inc.                         | Communication Services | Internet Content & Information | China     |    3.49e+09  |  16.96 |   28.14 |   0.0484 | 922812           |
+|  2 | JD       | JD.com, Inc.                          | Consumer Cyclical      | Internet Retail                | China     |    8.152e+10 | 300.85 |   53.25 |   0.0669 |      1.34395e+07 |
+|  3 | NI       | NiSource Inc.                         | Utilities              | Utilities - Regulated Gas      | USA       |    1.09e+10  |  17.69 |   27.23 |   0.0004 |      6.2667e+06  |
+|  4 | ZTO      | ZTO Express (Cayman) Inc.             | Industrials            | Integrated Freight & Logistics | China     |    1.947e+10 |  24.67 |   24.05 |   0.0217 |      3.06905e+06 |
+|  5 | QDEL     | QuidelOrtho Corporation               | Healthcare             | Diagnostics & Research         | USA       |    5.53e+09  |   4.96 |   83.32 |  -0.018  | 508575           |
+|  6 | ED       | Consolidated Edison, Inc.             | Utilities              | Utilities - Regulated Electric | USA       |    3.366e+10 |  20.21 |   96.3  |  -0.0025 |      1.3419e+06  |
+|  7 | SFM      | Sprouts Farmers Market, Inc.          | Consumer Defensive     | Grocery Stores                 | USA       |    3.57e+09  |  14.67 |   33.51 |  -0.0095 |      1.14442e+06 |
+|  8 | PINC     | Premier, Inc.                         | Healthcare             | Health Information Services    | USA       |    3.92e+09  |  21.52 |   33.2  |   0.0097 | 373171           |
+|  9 | AEP      | American Electric Power Company, Inc. | Utilities              | Utilities - Regulated Electric | USA       |    4.791e+10 |  19.63 |   94.65 |  -0.006  |      2.04444e+06 |
+| 10 | LRN      | Stride, Inc.                          | Consumer Defensive     | Education & Training Services  | USA       |    1.54e+09  |  16.74 |   35.8  |  -0.0047 | 270842           |
+| 11 | AMN      | AMN Healthcare Services, Inc.         | Healthcare             | Medical Care Facilities        | USA       |    5.4e+09   |  11.76 |  122.01 |  -0.0078 | 264138           |
+
+### Ownership
+
+When `data_type = 'ownership'`, data presented are statistics for the general float, insider, institutional, and the short ratio.
+
+openbb.stocks.screener.screener_data(preset_loaded='sdk_guide_preset.ini', data_type = 'ownership', limit = 100)
```

Comment:
This piece of code is missing a code block.

Reply:
Good catch.